src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,5 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
+    public static final Prefix PREFIX_REMARK = new Prefix("r/");
```

Comment:
Should we allow users to directly add people with remarks or edit remarks with the edit command if we have this prefix?